utils/activity.ts

```diff
@@ -1,13 +1,25 @@
 import { Context } from "@azure/functions";
 import * as t from "io-ts";
+import { EycaCard } from "../generated/definitions/EycaCard";
 
 // Activity result
 export const ActivityResultSuccess = t.interface({
   kind: t.literal("SUCCESS")
 });
-
 export type ActivityResultSuccess = t.TypeOf<typeof ActivityResultSuccess>;
 
+export const SuccessValue = t.union([t.undefined, EycaCard]);
```

Comment:
Since this is specific for EYCA implementation I'd rather move these definitions directly into `StoreEycaExpirationActivity`.

Reply:
Not sure about the `union` implementation. My idea was to have a generic `SuccessValue` to be returned by the Activity within an `ActivityResult` if necessary. 
How can I make this `SuccessValue` generic to match all the cases?

Reply:
You can't if you want a strictly typing :). Or even we have to map a Generic `SuccessValue` with other specific decoders that could fit our needs.

Reply:
Done in [#9d07ba5](https://github.com/pagopa/io-functions-cgn/pull/27/commits/9d07ba561352276df16d81ca4859d3d33e9c88fa)